packages/excalidraw/renderer/renderElement.ts

```diff
@@ -907,7 +908,8 @@ export const renderElement = (
           (!element.angle ||
             // or check if angle is a right angle in which case we can still
             // disable smoothing without adversely affecting the result
-            isRightAngle(element.angle))
+            // We need less-than comparison because of FP artihmetic
+            Math.abs(Math.sin(2 * element.angle)) < PRECISION)
```

Comment:
we should probably wrap this in a function like `isRightAngle` too, might not be too obvious for someone else reading this line